hubblestack/hec/opt.py

```diff
@@ -42,9 +49,10 @@ def _get_splunk_options(space, modality, **kw):
         'index_extracted_fields': [],
         'http_event_collector_ssl_verify': True,
         'add_query_to_sourcetype': True,
-        'disk_queue': False, # '/var/cache/hubble/dq',
-        'disk_queue_size': 10 * (1024 ** 2),
-        'disk_queue_compression': 5,
+        # disk_queue* can come from the top of the config
+        'disk_queue': confg('disk_queue', False),
+        'disk_queue_size': confg('disk_queue_size', 10 * (1024 ** 2)),
```

Comment:
Since the feature is disabled by default, can we make the default size setting be 100MB anyway?